src/config_file.rs

```diff
@@ -148,6 +148,14 @@ impl Config {
     }
 }
 
+impl Config {
```

Comment:
how about marking this `#[cfg(test)]` instead of `dead_code`

Reply:
Good call. Thanks.